attestation-service/verifier/src/se/mod.rs

```diff
@@ -0,0 +1,75 @@
+// Copyright (C) Copyright IBM Corp. 2024
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use super::*;
+use async_trait::async_trait;
+use base64::prelude::*;
+use serde_json::json;
+use crate::{InitDataHash, ReportData};
+use crate::se::seattest::FakeSeAttest;
+use crate::se::seattest::SeFakeVerifier;
+
+pub mod seattest;
+
+#[derive(Debug, Default)]
+pub struct SeVerifier {}
+
+#[async_trait]
+impl Verifier for SeVerifier {
+    async fn evaluate(
+        &self,
+        evidence: &[u8],
+        expected_report_data: &ReportData,
+        expected_init_data_hash: &InitDataHash,
+    ) -> Result<TeeEvidenceParsedClaim> {
+        verify_evidence(evidence, expected_report_data, expected_init_data_hash).await
+    }
+
+    async fn generate_supplemental_challenge(
+        &self,
+        _tee_parameters: Option<Vec<u8>>,
+    ) -> Result<String> {
+
+        // TODO replace FakeSeAttest with real crate
+        let attester = FakeSeAttest::default();
+
+        // TODO replace the placeholder
+        let hkds: Vec<String> = vec![String::new(); 2];
+        let certk = "cert_file_path";
+        let signk = "sign_file_path";
+        let arpk = "arpk_file_path";
+
+        let extra_params = attester.create(hkds, certk, signk, arpk)
```

Comment:
```suggestion
        let challenge = attester.create(hkds, certk, signk, arpk)
```

The rename matches the function name to avoid confusion.

Reply:
Good point, fixed.